src/app/fdctl/run/tiles/fd_dedup.c

```diff
@@ -163,7 +161,7 @@ after_frag( fd_dedup_ctx_t *    ctx,
     if( FD_UNLIKELY( !txn_t_sz ) ) FD_LOG_ERR(( "fd_txn_parse failed for vote transactions that should have been sigverified" ));
```

Comment:
Erroring out here is now dangerous, as unparsed transactions can now come from gossip, so there is no guarantee that the parsing will succeed.

We should probably either call `fd_txn_parse` inside the gossip tile (in `verify_vote_txn`?), or not error out here. I think the first option is probably better.